rrfs-test/validated_yamls/templates/obtype_config/satwnd_winds_245_g18.yaml

```diff
@@ -1,17 +1,18 @@
      - obs space:
          name: satwnd_winds_245_272
          distribution:
-           name: "RoundRobin"
-           halo size: 300e3
+           name: "@DISTRIBUTION@"
+           halo size: 500e3
          obsdatain:
            engine:
              type: H5File
-             #obsfile: data/obs/ioda_satwnd.nc
-             obsfile: "/scratch4/NCEPDEV/fv3-cam/Donald.E.Lippi/RRFSv2/ioda_processing/python/ioda_satwnd_245.abi_goes-18_dc_llp.nc"
+             obsfile: "data/obs/ioda_satwnd.nc"
+             missing file action: "warn"
          obsdataout:
+           empty obs space action: "@emptyObsSpaceAction@"
            engine:
              type: H5File
-             obsfile: jdiag_satwnd_winds_245_272.nc
+             obsfile: jdiag_satwnd_winds_245_g18.nc
```

Comment:
Suggest changing `jdiag_satwnd_winds_245_g18.nc` to `jdiag_satwnd_winds_245_272.nc` so as to be consistent with `obs space.name`. Also, changing the file name to make all are the same page. Thanks!